src/app/view/modules/object/object.component.ts

```diff
@@ -97,4 +104,39 @@ export class ObjectComponent implements OnChanges, OnInit {
     }
 
 
+    /*
+    * Requests incoming resources for the supplied ReadResourcesSequence, and adds them to the respective ReadResource.incoming array.
+    * Currently we only display the properties of this.resource.resources[0] (see html template),
+    * therefore we also request only the incoming resources to resourceSequence.resources[0].
+    *
+    * @param {ReadResourcesSequence} resourceSequence - the sequence of resources for which incoming resources should be requested.
+    */
+    private requestIncomingResources(resourceSequence: ReadResourcesSequence) {
+      if (resourceSequence.resources.length > 0) {
+
+        // request incoming Regions
+        if (resourceSequence.resources[0].properties[AppConfig.hasStillImageFileValue]) {
+          this._incomingService.getIncomingRegions(resourceSequence.resources[0].id).subscribe(
+            (result: ApiServiceResult) => {
+              let promise = jsonld.promises.compact(result.body, {});
+              promise.then((compacted) => {
+                let regions: ReadResourcesSequence = ConvertJSONLD.createReadResourcesSequenceFromJsonLD(compacted);
+                 Array.prototype.push.apply(resourceSequence.resources[0].incoming, regions.resources);
```

Comment:
I’m not sure, if you propose using:
 a) `resourceSequence.resources[0].incoming.push.apply(resourceSequence.resources[0].incoming, regions.resources)` 
or 
b) `resourceSequence.resources[0].push(regions.resources)`.

a) would be semantically equivalent to the current code, but is uglier to read, and i think the current code is making it more clear that we are using a static method of Array and not modifying `resourceSequence.resources[0].incoming`.
b) would insert the array `regions.resources` to the first element of `resourceSequence.resources[0].incoming`, which is not what we want. 
(Reassigning the whole `resourceSequence.resources[0].incoming` is also not what we want, if we consider that `requestIncomingResources()` might in the future request more incoming resource types which should all be added to `resourceSequence.resources[0].incoming` not replace it.)